src/components/Landing.jsx

```diff
@@ -1,9 +1,39 @@
 import React from "react";
+import { FaDiscord } from "react-icons/fa";
+import { AiFillInstagram } from "react-icons/ai";
+import { MdEmail } from "react-icons/md";
+import { FaLink } from "react-icons/fa";
+import { PiTiktokLogoBold } from "react-icons/pi";
 
 const Landing = () => {
   return (
-    <div>
-      <p>Landing</p>
+    <div className="flex flex-col items-center">
+      <p className="text-art-pink-200 text-lg font-bold ">Get Involved</p>
+      <hr className="border-2 border-art-pink-200 w-12" />
+      <div className="flex py-5 space-x-10">
+        <div className="flex flex-col items-center">
+          <FaDiscord size="48px" color="#E893C5" />
```

Comment:
too many divs move turn this into a  .map() to make it more cleaner since you essentially reusing (flex flex-col ) and create a socials.js file in the data folder